bindings/nostr-ffi/src/contact.rs

```diff
@@ -23,6 +24,7 @@ impl Deref for Contact {
 impl Contact {
     pub fn new(pk: String, relay_url: Option<String>, alias: Option<String>) -> Result<Self> {
         let pk = XOnlyPublicKey::from_str(&pk)?;
+        let relay_url = relay_url.map(|u| UncheckedUrl::from_str(&u).unwrap());
```

Comment:
Can you remove the `unwrap`?

I prefer to avoid the usage of `unwrap` and `except` in my libraries (in examples or tests are okay).

Will not be elegant as your solution but in this way the error is propagated:

```rust
 let relay_url = match relay_url {
    Some(relay_url) => Some(UncheckedUrl::from_str(&relay_url)?),
    None => None,
};
```

If you have another solution, feel free to use that (the important thing is that not use `unwrap` or `except`)

Reply:
I only thought i could get away with it in this case because since it can't really fail as its only wrapping the string but better to be consistent.